docs/content/developer/getting-started/live-concert.mdx

```diff
@@ -0,0 +1,241 @@
+```
+
+### [`resale`](https://github.com/iota-community/on-chain-live-concerts-ticketing-system/blob/3e902862a27886231a1721ab3609100326d284ad/package/sources/live_concert.move#L150-L164)
+
+- This function allows users to resell their tickets.
+- Users can also set the price at which they want to resell the ticket.
+- The `resale` function then creates a [`shared_object`](../iota-101/objects/object-ownership/shared.mdx) type objects `InitiateResale` in which the NFT and its resell price will be stored.
+
+```move reference
+https://github.com/iota-community/on-chain-live-concerts-ticketing-system/blob/3e902862a27886231a1721ab3609100326d284ad/package/sources/live_concert.move#L150-L164
+```
+
+### [`buy_resale_ticket`](https://github.com/iota-community/on-chain-live-concerts-ticketing-system/blob/3e902862a27886231a1721ab3609100326d284ad/package/sources/live_concert.move#L167-L187)
+
+- This function allows users to buy the tickets that are on resale.
+- Users will pass the address of the `InitiateResale` object that will store the NFT users want to buy.
+- The function then unwraps the NFT from the shared object and transfers it to the sender.
+- The `InitiateResale` object will be deleted, and the price of reselling the NFT will be transferred to its previous owner.
+
+```move reference
+https://github.com/iota-community/on-chain-live-concerts-ticketing-system/blob/3e902862a27886231a1721ab3609100326d284ad/package/sources/live_concert.move#L167-L187
+```
+
+### [`change_price`](https://github.com/iota-community/on-chain-live-concerts-ticketing-system/blob/3e902862a27886231a1721ab3609100326d284ad/package/sources/live_concert.move#L135-L148)
+
+- Only the creator of the package can call this function.
+- This function allows the creator to change the price of a category, by updating the `Category` object.
+
+```move reference
+https://github.com/iota-community/on-chain-live-concerts-ticketing-system/blob/3e902862a27886231a1721ab3609100326d284ad/package/sources/live_concert.move#L135-L148
+```
+
+:::note Helper Functions
+	There are some helper functions in the package that are used by the above functions.
+:::
+
+### Write Unit Tests
+
+After creating the module, you should write the tests for all the functions. Copy the below code to the `live_concert_tests.move` file:
+
+```move reference
+https://github.com/iota-community/on-chain-live-concerts-ticketing-system/blob/3e902862a27886231a1721ab3609100326d284ad/package/tests/live_concert_tests.move
+```
+
+### Publish the Package
+
+
+[Publish](./publish.mdx) the package to the IOTA Testnet using the following command:
+
+```bash 
+iota client publish
+```
+
+## Frontend dApp
+
+Next, let's create the user interface for the live concert ticketing system. To build the frontend dApp, we will use the [IOTA dApp kit](../../../ts-sdk/dapp-kit/).
+
+### Set Up
+
+Run the following command to set up the initial app:
+
+```bash
+npm create @iota/create-dapp
```

Comment:
npm error code E404
npm error 404 Not Found - GET https://registry.npmjs.org/@iota%2fcreate-create-dapp - Not found

Reply:
```suggestion
npm create @iota/dapp
```